website/docs/reference/node-selection/graph-operators.md

```diff
@@ -20,17 +20,17 @@ to step through.
 
 
   ```bash
-dbt run --select "my_model+1"        # select my_model and its first-degree children
-dbt run --select "2+my_model"        # select my_model, its first-degree parents, and its second-degree parents ("grandparents")
-dbt run --select "3+my_model+4"      # select my_model, its parents up to the 3rd degree, and its children down to the 4th degree
+dbt run --select "my_model+1"        # select my_model and its first-degree descendants
+dbt run --select "2+my_model"        # select my_model, its first-degree ancestors ("parents", and its second-degree ancestors ("grandparents")
+dbt run --select "3+my_model+4"      # select my_model, its ancestors up to the 3rd degree, and its descendants down to the 4th degree
   ```
 
 
 ### The "at" operator
-The `@` operator is similar to `+`, but will also include _the parents of the children of the selected model_. This is useful in continuous integration environments where you want to build a model and all of its children, but the _parents_ of those children might not exist in the database yet. The selector `@snowplow_web_page_context` will build all three models shown in the diagram below.
+The `@` operator is similar to `+`, but will also include _the ancestors of the descendants of the selected model_. This is useful in continuous integration environments where you want to build a model and all of its descendants, but the _ancestors_ of those descendants might not exist in the database yet. The selector `@snowplow_web_page_context` will build all three models shown in the diagram below.
```

Comment:
```suggestion
The `@` operator is similar to `+`, but will also include _all ancestors of the descendants of the selected model_. This is useful in continuous integration environments where you want to build a model and all of its descendants, but the _ancestors_ of those descendants might not exist in the schema yet. The `@` operator will select as many degrees of ancestors ("parents," "granparents," and so on) as is needed to successfully build the descendants. The selector `@snowplow_web_page_context` will build all three models shown in the diagram below.
```

Reply:
Thx for this clarification @foundinblank !